core-bundle/src/Routing/ResponseContext/JsonLd/ContaoPageSchema.php

```diff
@@ -98,6 +99,14 @@ public function getGroups(): array
         return $this->properties['groups'];
     }
 
+    /**
+     * @return array<int>
+     */
+    public function getCurrentGroups(): array
+    {
+        return $this->properties['currentGroups'];
```

Comment:
Can we rename this to `userGroups` or `memberGroups`? Otherwise there is `groups` and `currentGroups` and it is not clear what "current" refers to IMHO.

Reply:
`groups` represents the groups the page is assigned to. The new property represents the groups of the currently logged in member. It's only shown when you are logged in anyway so not sure it should be even part of the schema.org data. I just put it here because the rest is also here that doesn't really make sense (e.g. `fePreview`). A HTTP header would've been probably the better choice.
But `memberGroups` or `userGroups` still doesn't make it clear that it's the ones of the currently logged in member - does it?

Reply:
Not 100%, I agree. But `memberGroups` at least suggests that the groups belong to a member.

Reply:
Sure, updated.